lib/plugins/aws/lib/naming.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = {
+  /*
+   * General Name Normalization
+   */
+  normalizeName(name) {
+    return `${_.upperFirst(name)}`;
+  },
+  normalizeNameToAlphaNumericOnly(name) {
+    return this.normalizeName(name.replace(/[^0-9A-Za-z]/g, ''));
+  },
+  normalizePathtoCapitalAlphaNumbericOnlyWithReplacement(path) {
+    return this.normalizeNameToAlphaNumericOnly(
+      path.replace(/-/g, 'Dash')
+        .replace(/\{(.*)\}/g, '$1Var'));
+  },
+
+  getServiceEndpointRegex() {
+    return /^ServiceEndpoint/;
+  },
+
+  /*
```

Comment:
Is there a need to have multi line comments instead of a single line comment?

Would make grasping the content of the file a little bit easier and reduces scrolling IMHO.